ts/src/bitmex.ts

```diff
@@ -226,29 +226,33 @@ export default class bitmex extends Exchange {
                 'networks': {
                     'BTC': 'btc',
                     'ETH': 'eth',
-                    'BSC': 'bsc',
-                    'BNB': 'bsc',
-                    'TRON': 'tron',
                     'ERC20': 'eth',
                     'BEP20': 'bsc',
+                    'BSC': 'bsc',
                     'TRC20': 'tron',
+                    'TRON': 'tron',
                     'TRX': 'tron',
-                    'AVAX': 'avax',
+                    'AVALANCHEC': 'avax',
```

Comment:
that is our unified networkcode for avalanche-chain-**c** (as opposed to avalanche-chain-**x**)